### DISCUSSION TITLE
DataFusion twitter / X account management

### DISCUSSION THREAD

**Discussion by alamb** [apache/datafusion#13094] [General]
There is a DataFusion twitter account: https://x.com/ApacheDataFusio

(yes I know it doesn't end with `n`, because of the twitter name limit`

I made this account and use it occasionally to promote / retweet mentions, but don't really spend much time using it. 

Recently @SamSynnada offered to help run it more actively (promote DataFusion specific content, etc):
-  https://github.com/apache/datafusion/discussions/13049

Are there any thoughts about giving @SamSynnada access to the account to help?

**Comment by matthewmturner** [comment]
Are there any guidelines from Apache on how to handle these types of decisions? For example does it need to be voted on by PMC?   Normally I would have hesitation about giving someone access who is relatively new to the project, but given he is a cofounder of Synnada, who have multiple long time contributors, i think its actually great.

Separately, that missing `n` really is unfortunate, it makes the account not seem official.

**Reply by alamb** [reply]
https://www.apache.org/foundation/marks/socialmedia#access appears to be quite relevant

**Comment by comphead** [comment]
Thanks @alamb that sounds good to me as it increases the visibility and expands to the audience.
Before making any thoughts on X/Twitter please help to understand what kind of content is planned to broadcast there? If I understand correctly from #13049 it is not very technical data, sort of news, project life, evolvement and accomplishments. That sounds incredibly useful, perhaps we can also have a group in Linkedin like Apache Spark does https://www.linkedin.com/company/apachespark/

**Reply by alamb** [reply]
My understanding is that https://github.com/apache/datafusion/discussions/13049  proposes using the twitter account to highlight specific datafusion things (e.g link to datafusion content)

Maybe it would be a good idea to document what the expectations on content would be

**Comment by andygrove** [comment]
Could we also include Bluesky in this discussion. There is a rapidly growing data community there now.

**Reply by alamb** [reply]
What / who is bluesky? I am not familiar with that

**Reply by andygrove** [reply]
https://en.wikipedia.org/wiki/Bluesky

Bluesky just reached critical mass with the tech/data community moving over there in the past few days. DuckDB just created an account there for example.

**Comment by andygrove** [comment]
I created a Blyesky account: https://bsky.app/profile/apachedatafusion.bsky.social

I used private@datafusion.apache.org as the registered email address so that any PMC member can reset the password.

**Comment by andygrove** [comment]
I am +1 for the proposal, but we cannot make any decisions in a GitHub discussion because it is not mirrored on any mailing list.

@alamb Should we convert this discussion into a GitHub issue?

**Reply by alamb** [reply]
Yes thank you

**Reply by andygrove** [reply]
Here is the issue: https://github.com/apache/datafusion/issues/13178

**Comment by alamb** [comment]
Let's move the discussion to the issue so we have a record
